lightkurve/search.py

```diff
@@ -0,0 +1,460 @@
+from __future__ import division, print_function
+import os
+import logging
+import numpy as np
+
+from astroquery.mast import Observations
+from astroquery.exceptions import ResolverError
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+from astropy.io import ascii, fits
+from .lightcurve import KeplerLightCurve, TessLightCurve
+from .lightcurvefile import KeplerLightCurveFile
+from .targetpixelfile import TargetPixelFile, KeplerTargetPixelFile
+from .collections import TargetPixelFileCollection, LightCurveCollection, LightCurveFileCollection
+
+from . import PACKAGEDIR
+log = logging.getLogger(__name__)
+
+class SearchResult(object):
+    """
+
+    """
+
+    def __init__(self, path, campaign=None, quarter=None, month=None, cadence=None):
+
+        self.path = path
+        self.campaign = campaign
+        self.quarter = quarter
+        self.month = month
+        self.cadence = cadence
+
+    @property
+    def info(self):
+        return self._make_table(self.path)
+
+    @property
+    def obsID(self):
+        return np.asarray(np.unique(self.path['obsid']), dtype='int')
+
+    @property
+    def target_name(self):
+        return np.asarray(np.unique(self.path['target_name']))
+
+    @property
+    def RA(self):
```

Comment:
We've been using lowercase `ra` elsewhere so we'll probably want to stick to that.